internal/pkg/cli/svc_override.go

```diff
@@ -73,6 +98,35 @@ func (o *overrideSvcOpts) RecommendActions() error {
 	return nil
 }
 
+func (o *overrideSvcOpts) validateAppName() error {
+	if o.appName == "" {
+		return errNoAppInWorkspace
+	}
+	if _, err := o.appConfig(); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (o *overrideSvcOpts) validateCDKLang() error {
+	for _, valid := range validCDKLangs {
+		if o.cdkLang == valid {
+			return nil
+		}
+	}
+	return fmt.Errorf("%q is not a valid CDK language: must be one of: %s",
+		o.cdkLang,
+		strings.Join(applyAll(validCDKLangs, strconv.Quote), ", "))
+}
+
+func (o *overrideSvcOpts) appConfig() (*config.Application, error) {
```

Comment:
Are we going to use this method somewhere else if not then just keeping return type as error would be fine I think. We do not need to return `config.Application` if we aren't planning to use it.

Reply:
Sure, I removed the `appConfig` method and inlined it instead